ImaguruClassWork/ImaguruClassWork/Modules/Feed/View/FeedView.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  FeedView.swift
+//  ImaguruClassWork
+//
+//  Created by Wishell on 01.11.2018.
+//  Copyright © 2018 Wishell. All rights reserved.
+//
+
+import UIKit
+
+protocol FeedViewInput: class {
+    var registrateTable: ((UITableView) -> Void)? { get set }
+    func prepareTable()
+
+    var tableDataSource: (() -> DataSource)? { get set }
+    var onTableItemTap: ((Article) -> Void)? { get set }
+    var onExitTapAction: (() -> Void)? { get set }
+}
+
+final class FeedView: UIView {
+    @IBOutlet weak var tableView: UITableView!
+    var registrateTable: ((UITableView) -> Void)?
+    var tableDataSource: (() -> DataSource)?
+    var onTableItemTap: ((Article) -> Void)?
+    var onExitTapAction: (() -> Void)?
+    @IBAction func onExitTap(_ sender: Any) {
```

Comment:
separate Outlets, Actions and protocol conformance (use new line as separator)

final class FeedView: UIView {

    @IBOutlet weak var tableView: UITableView!

    var registrateTable: ((UITableView) -> Void)?
    var tableDataSource: (() -> DataSource)?